module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+#include <cstdint>
```

Comment:
Why are you including cstdint in such weird way(in another paragraph) instead of putting all of includes alphabetically?

Reply:
Don't know :) we haven't got much time  last week